var/spack/repos/builtin/packages/brpc/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Brpc(CMakePackage):
+    """An industrial-grade RPC framework used throughout Baidu, with
+    1,000,000+ instances(not counting clients) and thousands kinds of
+    services, called "baidu-rpc" inside Baidu. Only C++ implementatioo
+    on is opensourced right now."""
+
+    homepage = "https://github.com/apache/incubator-brpc"
+    url      = "https://github.com/apache/incubator-brpc/archive/0.9.7-rc03.tar.gz"
+
+    version('0.9.7-rc03', sha256='baacaed6c1dcdda06cb686fc4a5218da3649b9727436288d0369a68d475e1d93')
```

Comment:
We don't normally add release candidates of packages. It looks like 0.9.7 is out now, can you add that instead?

Reply:
I will update some stable version.
But `0.9.7-rc03` is `0.9.7` and its url is : `https://github.com/apache/incubator-brpc/archive/0.9.7-rc03.tar.gz`
So I think we may keep this version?

Reply:
Huh, you're right, that is really weird. I wonder if that was a mistake?

Reply:
I'm disscussing this issue with upstream at here: https://github.com/apache/incubator-brpc/issues/1103

Reply:
This issue is settle down by upstream, and I update the version in this patch: 
https://github.com/spack/spack/pull/16309/commits/20aedc65decef1bf7bbee64da3a4ff60f161bab4